src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -122,23 +123,28 @@
     },
     
     /*******************************************************************************************************
-    * @description Saves all changes and return to the household
+    * @description Saves all changes the container knows about (household object/account, contacts), and
+    * if successful, then calls the visualforce page's save actionMethod, which will also close the page.
     */
     save : function(component) {
         component.set("v.showSpinner", true);
         
+		// keep track of ourselves and whether both server updates succeed
+        var self = this;
+        var countSuccesses = 0;
+
         // save the contacts
         var listCon = component.get("v.listCon");
 		var action = component.get("c.updateContacts");
         action.setParams({ listCon : listCon});
-		var self = this;
-        var countSuccesses = 0;
 		action.setCallback(this, function(response) {
 			var state = response.getState();
 			if (component.isValid() && state === "SUCCESS") {
                 countSuccesses++;
-                if (countSuccesses == 2)
-	                self.close(component);
+                if (countSuccesses == 2) {
+                    saveVisualforce();
```

Comment:
Since this method is declared outside of the LC, then if this LC is used on any page that doesn't also declare this method, then this call will fail.  This means the HH_Container LC is not generally re-useable.  This may be okay, depending on intended usage.  What do you think, @davidhabib?